AsyncDisplayKit/TextKit/ASTextKitRenderer.mm

```diff
@@ -176,11 +183,32 @@ - (void)drawInContext:(CGContextRef)context bounds:(CGRect)bounds;
   LOG(@"%@, shadowInsetBounds = %@",self, NSStringFromCGRect(shadowInsetBounds));
   
   [[self context] performBlockWithLockedTextKitComponents:^(NSLayoutManager *layoutManager, NSTextStorage *textStorage, NSTextContainer *textContainer) {
+    
+    NSTextStorage *scaledTextStorage = nil;
+    BOOL isScaled = (self.currentScaleFactor > 0 && self.currentScaleFactor < 1.0);
+
+    if (isScaled) {
```

Comment:
Oops, I highlighted the wrong section. The part I wasn't sure about was swapping out the layout manager's text storage right before render.